src/test/java/com/puppycrawl/tools/checkstyle/bdd/InputConfiguration.java

```diff
@@ -32,57 +32,63 @@
     /** The check name. */
     private final String checkName;
 
-    /** Map of default properties. */
-    private final Map<String, String> defaultProperties;
+    /** Map of default attributes. */
+    private final Map<String, String> defaultAttributes;
```

Comment:
there is no Atributes in Checkstyle, we have properties.
https://checkstyle.org/config.html#Properties

restore original names.

Reply:
we have attributes here `DefaultConfiguration.getAttribute` :) 
It seems more natural to have same naming for `InputConfiguration` and `DefaultConfiguration`

Reply:
@romani ping

Reply:
https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/DefaultConfiguration.java

It is not api class, better to rename in such class also. In separate issue

Reply:
@shashwatj07 lets move back to properties then

Reply:
> @shashwatj07 lets move back to properties then

done

Reply:
> https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/DefaultConfiguration.java
> 
> It is not api class, better to rename in such class also. In separate issue

opened https://github.com/checkstyle/checkstyle/issues/10164